src/inc/readytorun.h

```diff
@@ -180,6 +180,8 @@ enum ReadyToRunFixupKind
 
     READYTORUN_FIXUP_DelegateCtor               = 0x2C, /* optimized delegate ctor */
     READYTORUN_FIXUP_DeclaringTypeHandle        = 0x2D,
+
+    READYTORUN_FIXUP_IndirectPInvokeTarget      = 0x56, /* Target of an inlined pinvoke */
```

Comment:
Why is this 0x56 and not the next available number in the sequence?

Reply:
Because it has to match ENCODE_INDIRECT_PINVOKE_TARGET, which the Zap node encodes for these pinvokes (verified by the static assert)

Reply:
It does not need to match. The fragile NGen specific fixups were intentionally in the 0x50+ range so that they do not pollute R2R fixups. You should move this fixup to the R2R range and give it next number in the sequence.

Reply:
Sounds good, i'll look into it and make sure we use the R2R fixup number instead when writing signatures in R2R images.